### PR TITLE
Add demonstration example with no OS (bare metal).

### PR DESCRIPTION
**Pull request by ioannis-karachalios**
Changes to be committed:
	new file:   helpers/hello_world_no_os/.cproject
	new file:   helpers/hello_world_no_os/.project
	new file:   helpers/hello_world_no_os/README.md
	new file:   helpers/hello_world_no_os/config/custom_config_oqspi.h
	new file:   helpers/hello_world_no_os/config/custom_config_ram.h
	new file:   helpers/hello_world_no_os/main.c
	new file:   helpers/hello_world_no_os/makefile.targets

### CODE REVIEW

File: helpers/hello_world_no_os/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+|  Setting  | Value  |
+| :-------: | :----: |
+| Baud rate | 115200 |
+| Data bits |   8    |
+| Stop bits |   1    |
+|  Parity   |  None  |
+
+### Software configuration
+
+- Recommended to download the latest Smart Snippets Studio (SSS) version.
```
Comment (ibenamor) [inline helpers/hello_world_no_os/README.md]: Provide link: https://www.renesas.com/eu/en/software-tool/smartbond-development-tools

File: helpers/hello_world_no_os/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- Download the latest SDK version.
```
Comment (ibenamor) [inline helpers/hello_world_no_os/README.md]: Provide link: https://www.renesas.com/us/en/document/sws/da1470x-sdk-102649

File: helpers/hello_world_no_os/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- SEGGER J-Link tools should be downloaded and installed.
+
+## How to run the example
+
+**The RAM build configurations should be used for debugging purposes only.**
+
+### Initial Setup
+
+- Download the source code from the Support Website.
```
Comment (ibenamor) [inline helpers/hello_world_no_os/README.md]: Which source code ?

File: helpers/hello_world_no_os/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+- Import the `hello_world_no_os` sample code into your workspace (there are no path dependencies).
+
+- Build the project by selecting either the debug or release OQSPI or RAM build configuration.
+
+- Download the firmware image into the flash memory used by selecting the `program_oqspi_jtag` or  `program_oqspi_serial` launcher. If more actions are required follow the instructions given in the Console window of the Eclipse environment.
```
Comment (ibenamor) [inline helpers/hello_world_no_os/README.md]: may be here you can refer to the https://lpccs-docs.renesas.com/um-b-153-da1470x_getting_started/9-First_DA1470x_application/First_DA1470x_application.html section 9.3

File: helpers/hello_world_no_os/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+  - The project is compiled with the flash memory autodetect feature enabled meaning that any supported OQSPI flash memory can be used without modifying the `custom_config_oqspi.h` file, that is:
```
Comment (ibenamor) [inline helpers/hello_world_no_os/README.md]: File location: config/custom_config_oqspi.h

File: helpers/hello_world_no_os/main.c [hunk 6]
```diff
@@ -0,0 +1,35 @@
+/**
```
Comment (ibenamor) [inline helpers/hello_world_no_os/main.c]: should be The MIT License (MIT) - check the with @koen-vdm

File: helpers/hello_world_no_os/config/custom_config_oqspi.h [hunk 7]
```diff
@@ -0,0 +1,58 @@
+/**
```
Comment (ibenamor) [inline helpers/hello_world_no_os/config/custom_config_oqspi.h]: MIT License (MIT)

File: helpers/hello_world_no_os/config/custom_config_ram.h [hunk 8]
```diff
@@ -0,0 +1,58 @@
+/**
+ ****************************************************************************************
+ *
+ * @file custom_config_ram.h
```
Comment (ibenamor) [inline helpers/hello_world_no_os/config/custom_config_ram.h]: The MIT License (MIT)

**Review by ibenamor — CHANGES_REQUESTED** [review]

File: helpers/hello_world_no_os/config/custom_config_ram.h (hunk 8 above)
Comment (ioannis-karachalios) [reply helpers/hello_world_no_os/config/custom_config_ram.h:4]: Could you provide more info on this comment? I do not seen such 'contents' in other SDK files.

File: helpers/hello_world_no_os/config/custom_config_oqspi.h (hunk 7 above)
Comment (ioannis-karachalios) [reply helpers/hello_world_no_os/config/custom_config_oqspi.h:1]: See comment above.

File: helpers/hello_world_no_os/main.c (hunk 6 above)
Comment (ioannis-karachalios) [reply helpers/hello_world_no_os/main.c]: Could you provide more info on this comment? I do not see other SDK files containing that license comments.

File: helpers/hello_world_no_os/README.md (hunk 3 above)
Comment (ioannis-karachalios) [reply helpers/hello_world_no_os/README.md]: The underlying demonstration example.

Comment (ibenamor) [reply helpers/hello_world_no_os/README.md]: Sorry, Not clear to me what do you mean by "Download the demonstration example from the Support Website."

File: helpers/hello_world_no_os/main.c (hunk 6 above)
Comment (ibenamor) [reply helpers/hello_world_no_os/main.c]: the SW example should be released under the MIT license / check the SDK 10 SW example/ update license and copyright Pull request.

Comment (ioannis-karachalios) [reply helpers/hello_world_no_os/main.c]: Understood.
https://bitbucket.global.renesas.com/projects/LPCCSE/repos/sdk10_da1469x_examples/browse/LICENSE.txt

File: helpers/hello_world_no_os/README.md (hunk 3 above)
Comment (ioannis-karachalios) [reply helpers/hello_world_no_os/README.md]: Under the 'Documentation' section we used to have a link to our GitHub SW examples repo. It seems that this link is no longer available so better to state where the example can be downloaded from and provide a link to our GitHub repo.

**Comment by ioannis-karachalios** [comment]
@ibenamor, all the changes requested are fixed.

**Comment by ioannis-karachalios** [comment]
> @ibenamor, all the changes requested are fixed.

@ibenamor kind reminder to accept the PR so it can be merged into main.

**Review by ibenamor — APPROVED** [review]

**Pull request merged**